src/public/javascripts/ssoAuthES6.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See full license in root of repo. 
+ *
+ * This file shows how to use the SSO API to get a bootstrap token.
+ */
+
+Office.onReady(function(info) {
+    $(document).ready(function() {
+        $('#getGraphDataButton').click(getGraphData);
+    });
+});
+
+let retryGetAccessToken = 0;
+
+async function getGraphData() {
+    try {
+        let bootstrapToken = await OfficeRuntime.auth.getAccessToken({ allowSignInPrompt: true }); 
+        let exchangeResponse = await getGraphToken(bootstrapToken);
+        if (exchangeResponse.claims) {
+            // Microsoft Graph requires an additional form of authentication. Have the Office host 
+            // get a new token using the Claims string, which tells AAD to prompt the user for all 
+            // required forms of authentication.
+            let mfaBootstrapToken = await OfficeRuntime.auth.getAccessToken({ authChallenge: exchangeResponse.claims });
+            exchangeResponse = await getGraphToken(mfaBootstrapToken);
+        }
+        
+        if (exchangeResponse.error) {
+            // AAD errors are returned to the client with HTTP code 200, so they do not trigger
+            // the catch block below.
+            handleAADErrors(exchangeResponse);            
+        } 
+        else 
+        {
+            // For debugging:
+            // showMessage("ACCESS TOKEN: " + JSON.stringify(exchangeResponse.access_token));
```

Comment:
Remove comment for debugging.

Reply:
Given how much Edge DevTools from crashing, I thought this would be a valuable hint to devs that are using the sample. Do you not think so?